build.gradle

```diff
@@ -366,11 +452,66 @@ task apiJar(type: Jar) {
     getArchiveClassifier().set('api')
 }
 
-
 artifacts {
     archives sourcesJar
     archives devJar
     if(apiPackage) {
         archives apiJar
     }
 }
+
+// Updating
+task updateBuildScript {
+    doLast {
+        if (updateBuildScript()) return
+
+        print("Build script already up-to-date!")
+    }
+}
+
+if (isNewBuildScriptVersionAvailable()) {
+    if (autoUpdateBuildScript.toBoolean()) {
+        updateBuildScript()
+    } else {
+        println("Build script update available! Run 'gradle updateBuildScript'")
+    }
+}
+
+static URL availableBuildScriptUrl() {
+    new URL("https://raw.githubusercontent.com/SinTh0r4s/ExampleMod1.7.10/main/build.gradle")
```

Comment:
shouldn't the GTNH mods depend on a GTNH fork of this examplemod?

Reply:
Once it is done, yes. Right now we might change a lot of things